lib/RequestManager.ts

```diff
@@ -1,14 +1,13 @@
 import {HudAiClientConfiguration} from './util/ClientConfigFactory';
-import * as Request from 'request-promise';
 import * as _ from 'lodash';
 import {HudAiError} from './util/HudAiError';
 import {Session} from './Session';
 import * as Promise from 'bluebird';
+import Axios from 'axios';
```

Comment:
you're going to want to update the ClientConfigFactory defaults

Reply:
fixed